INSTALLATION.md

```diff
@@ -107,7 +107,44 @@ conda activate sf
 pip install -U secretflow
 ```
 
+## GPU support
+
+NVIDIA's CUDA and cuDNN are typically used to accelerate the training and testing of tensoflow and pytorch deep learning models. tensoflow and pytorch are both deep learning backends for secretflow, and if you want to use GPU acceleration in secretflow, you can follow these steps:
```

Comment:
`tensoflow` -> `TensorFlow` and `pytorch` -> `PyTorch`  are more official， same as `secretflow` -> `SecretFlow`.